build/bin/entrypoint

```diff
@@ -1,12 +1,3 @@
 #!/bin/sh -e
 
-# This is documented here:
```

Comment:
Can we remove this file or the operator-sdk leaves it empty?

Reply:
The operator-sdk does not leave it empty, and neither do we